frontend/packages/noobaa-storage-plugin/integration-tests/views/obcPage.view.ts

```diff
@@ -0,0 +1,109 @@
+import * as sideNavView from '@console/internal-integration-tests/views/sidenav.view';
+import { $, by, browser, element, ExpectedConditions as until } from 'protractor';
+import {
+  clickKebabAction,
+  deleteResource,
+  resourceRowsPresent,
+  untilNoLoadersPresent,
+} from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { SECOND } from '@console/ceph-storage-plugin/integration-tests/utils/consts';
+import { ATTACH_TO_DEPLOYMENT, OBC_RESOURCE_PATH } from '../utils/consts';
+
+export const OBCPage = async () => {
```

Comment:
Just to be consistent with the below
```suggestion
export const gotoOBCPage = async () => {
```